common-tasks/dispvm.md

```diff
@@ -11,74 +11,93 @@ redirect_from:
 
 ![r1-open-in-dispvm-1.png](/attachment/wiki/DisposableVms/r1-open-in-dispvm-1.png) ![r1-open-in-dispvm-2.png](/attachment/wiki/DisposableVms/r1-open-in-dispvm-2.png)
 
 Opening a fresh web browser instance in a new Disposable VM
 -----------------------------------------------------------
 
-Sometimes it is convenient to open a fresh instance of Firefox within a new fresh Disposable VM. This can be easily done by using the Start Menu: just go to Start -\> System Tools -\> DispVM:Firefox web browser . Wait a few seconds until a web browser starts. Once you close the viewing application the whole Disposable VM will get destroyed.
+Sometimes it is desirable to open an instance of Firefox within a new fresh Disposable VM. 
+This can be done easily using the Start Menu: just go to Start -\> System Tools -\> DispVM:Firefox web browser. 
+Wait a few seconds until a web browser starts. 
+Once you close the viewing application the whole Disposable VM will get destroyed. 
 
 ![r1-open-in-dispvm-3.png](/attachment/wiki/DisposableVms/r1-open-in-dispvm-3.png)
 
 Opening a file in a Disposable VM via command line (from AppVM)
 ---------------------------------------------------------------
 
-Use the `qvm-open-in-dvm` command line (from your AppVM), e.g.:
+Use the `qvm-open-in-dvm` command from your AppVM:
 
 ~~~
 [user@work-pub ~]$ qvm-open-in-dvm Downloads/apple-sandbox.pdf
 ~~~
 
-The qvm-open-in-dvm will not exit until you close the application in the Disposable VM.
+Note that the `qvm-open-in-dvm` process will not exit until you close the application in the Disposable VM.
 
 Starting an arbitrary application in a disposable VM via command line (from Dom0)
 ---------------------------------------------------------------------------------
 
-**Note:** Normally there should be no need for doing this -- this is just for Qubes hackers ;)
+**Note:** Normally there should be no need for doing this -- this is just for Qubes hackers ;). 
+It generally makes little sense to start any application other than a web browser this way.
 
 ~~~
 [joanna@dom0 ~]$ echo xterm | /usr/lib/qubes/qfile-daemon-dvm qubes.VMShell dom0 DEFAULT red
 ~~~
 
-In fact the Disposable VM appmenu used for starting Firefox contains a very similar command to the above. Please note, however, that it generally makes little sense to start any other application other than a Web Browser this way...
+(The Disposable VM appmenu used for starting Firefox contains a very similar command to the above.)
 
 Starting an arbitrary program in a Disposable VM from an AppVM
 --------------------------------------------------------------
 
-Sometimes it might be useful to start an arbitrary program, such as e.g. terminal in an Disposable VM from an AppVM. This could be simply done this way:
+Sometimes it can be useful to start an arbitrary program in a DispVM. This can be done from an AppVM by running
 
 ~~~
-[user@vault ~]$ qvm-run '$dispvm' xterm
+[user@vault ~]$ qvm-run '$dispvm' nano
```

Comment:
Why change this? `xterm` is always installed in Qubes templates, while `nano` is not.

Reply:
Ah I got mixed up. Will change back.